source/aggregation.txt

```diff
@@ -161,6 +186,156 @@ from the preceding :ref:`php-aggregation-example`:
+
+.. code-block:: php
+
+   $pipeline = [
+       Stage::<operator method>(
+           <stage specification>
+       ),
+       Stage::<operator method>(
+           <stage specification>
+       ),
+       ...
+   ];
+   
+   $cursor = $collection->aggregate($pipeline);
+
+The examples in this section are adapted from the {+mdb-server+} manual.
+Each example provides a link to the sample data that you can insert into
+your database to test the aggregation operation.
+
+Filter and Group Example
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+This example uses the sample data given in the :manual:`Calculate Count,
+Sum, and Average </reference/operator/aggregation/group/#calculate-count--sum--and-average>`
+section of the ``$group`` stage reference in the Server manual.
+
+The following code example calculates the total sales amount, average
+sales quantity, and sale count for each day in the year 2014. To do so,
+it uses an aggregation pipeline that contains the following stages:
+
+- :manual:`$match </reference/operator/aggregation/match/>` stage to
+  filter for documents that contain a ``date`` field in which the year is
+  2014
+
+- :manual:`$group </reference/operator/aggregation/group/>` stage to
+  group the documents by date and calculate the total sale amount,
+  average quantity, and total count for each group
+  
+- :manual:`$sort </reference/operator/aggregation/sort/>` stage to
+  sort the results by the total sale amount for each group in descending
+  order
+  
+.. io-code-block::
+   :copyable:
+
+   .. input:: /includes/aggregation.php
+      :start-after: start-builder-match-group
+      :end-before: end-builder-match-group
+      :language: php
+      :dedent:
+
+   .. output:: 
+      :visible: false
+
+      {"_id":"2014-04-04","totalSaleAmount":{"$numberDecimal":"200"},"averageQuantity":15,"count":2}
+      {"_id":"2014-03-15","totalSaleAmount":{"$numberDecimal":"50"},"averageQuantity":10,"count":1}
+      {"_id":"2014-03-01","totalSaleAmount":{"$numberDecimal":"40"},"averageQuantity":1.5,"count":2}
+
+Unwind Embedded Arrays Example
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This example uses the sample data given in the :manual:`Unwind Embedded Arrays
+</reference/operator/aggregation/unwind/#unwind-embedded-arrays>`
+section of the ``$unwind`` stage reference in the Server manual.
+
+The following code example groups sold items by their tags and
+calculates the total sales amount for each tag. To do so,
+it uses an aggregation pipeline that contains the following stages:
+
+- :manual:`$unwind </reference/operator/aggregation/unwind/>` stage to
+  output a separate document for each element in the ``items`` array
+
+- :manual:`$unwind </reference/operator/aggregation/unwind/>` stage to
+  output a separate document for each element in the ``items.tags`` arrays
+  
+- :manual:`$group </reference/operator/aggregation/group/>` stage to
+  group the documents by the tag value and calculates the total sales
+  amount of items that have each tag
```

Comment:
```suggestion
1. :manual:`$unwind </reference/operator/aggregation/unwind/>` stage to
  output a separate document for each element in the ``items`` array

#. :manual:`$unwind </reference/operator/aggregation/unwind/>` stage to
  output a separate document for each element in the ``items.tags`` arrays
  
#.  :manual:`$group </reference/operator/aggregation/group/>` stage to
  group the documents by the tag value and calculates the total sales
  amount of items that have each tag
```
S: Would do a numbered list instead, to show the order of the stages. Was also confused when I saw $unwind twice, so this might help to alleviate initial confusion. Would do for the other sections on the page as well.

Reply:
can change for all